packages/react-select/src/components/Group.tsx

```diff
@@ -115,19 +118,24 @@ export const groupHeadingCSS = <
   Option,
   IsMulti extends boolean,
   Group extends GroupBase<Option>
->({
-  theme: { spacing },
-}: GroupHeadingProps<Option, IsMulti, Group>): CSSObjectWithLabel => ({
+>(
+  { theme: { spacing } }: GroupHeadingProps<Option, IsMulti, Group>,
+  unstyled: boolean
+): CSSObjectWithLabel => ({
   label: 'group',
   color: '#999',
   cursor: 'default',
   display: 'block',
   fontSize: '75%',
   fontWeight: 500,
   marginBottom: '0.25em',
```

Comment:
I haven't (yet) looked to deeply into why some of these values are hardcoded, so i'm keeping them in the "permanent" section of the styles

Reply:
I think this was a mistake from when groups got added, they should be using theme variables. If you can line them up with existing theme values, please do 🙏